server/locale/US/Pi4/styles/flex/ratio--20x1.css

```
@@ -0,0 +1,117 @@
+@media (min-aspect-ratio: 200/11) {
+    .message__logo-container {
+        margin-bottom: -3px;
+        max-width: 12%;
+        margin-right: 1.5vw;
+    }
+
+    .message__logo:nth-of-type(1) {
+        width: 15%;
+        margin-right: 3%;
+    }
+
+    .message__logo:nth-of-type(2) {
+        width: 55%;
+    }
+
+    .message__promo-container {
+        width: unset;
+    }
+
+    .message__messaging {
+        flex: none;
+        display: inline;
+    }
+
+    .message__content {
+        justify-content: center;
+    }
+
+    .message__sub-headline {
+        margin-left: 0;
+    }
+
+    .message__disclaimer {
+        margin-left: 4px;
+    }
+}
+
+@media (min-aspect-ratio: 200/11) and (min-width: 351px) {
+    .message__disclaimer {
+        font-size: 0.8rem;
+    }
+}
+
```

New media query you can add on line 45.
```suggestion
@media (min-aspect-ratio: 200/11) and (min-width: 903px) {
    .message__logo-container {
        margin-bottom: -6px;
    }
}
```